repository_service_tuf_api/bootstrap.py

```diff
@@ -77,18 +77,7 @@ class TUFSigned(BaseModel):
     expires: str
     keys: Optional[Dict[str, TUFKeys]]
     consistent_snapshot: Optional[bool]
-    roles: Optional[
-        Dict[
-            Literal[
-                Roles.ROOT.value,
-                Roles.TARGETS.value,
-                Roles.SNAPSHOT.value,
-                Roles.TIMESTAMP.value,
-                Roles.BINS.value,
-            ],
-            TUFSignedRoles,
-        ]
-    ]
+    roles: Optional[Dict[Roles, TUFSignedRoles]]
```

Comment:
Please take a look at the problem I described below.
Here you have the same issue. Maybe it will be better not to use the Enum 🤔